### ISSUE TITLE
Tab content does not change in split screens when order is changed

### ISSUE THREAD

**Issue by cimbi** [spyder-ide/spyder#4941]
## Description

Changes in tab order do to propagate to split screened editor

**What steps will reproduce the problem?**

1. Split the editor widget into two (either horizontal or vertical)
2. Drag & drop the opened tabs - change their order in one split screen
3. The order of tabs changes in both screens but their content does not

**What is the expected output? What do you see instead?**
The content of the tabs gets changed along with tab labels.

**Please provide any additional information below**

## Version and main components

* Spyder Version: 3.2.0 
* Python Version: 3.5.2
* Qt Versions: 5.5.1, PyQt5 5.5.1 on Linux

## Dependencies
```
IPython >=4.0    :  6.1.0 (OK)
cython >=0.21    :  None (NOK)
jedi >=0.9.0     :  0.10.2 (OK)
nbconvert >=4.0  :  5.2.1 (OK)
numpy >=1.7      :  1.13.0 (OK)
pandas >=0.13.1  :  0.20.2 (OK)
psutil >=0.3     :  5.2.2 (OK)
pycodestyle >=2.3:  2.3.1 (OK)
pyflakes >=0.6.0 :  1.5.0 (OK)
pygments >=2.0   :  2.2.0 (OK)
pylint >=0.25    :  1.7.2 (OK)
qtconsole >=4.2.0:  4.3.0 (OK)
rope >=0.9.4     :  0.10.5 (OK)
sphinx >=0.6.6   :  1.6.3 (OK)
sympy >=0.7.3    :  1.0 (OK)

```

**Comment by stonebig** [comment]
Aren't your issue similar to this one see https://github.com/spyder-ide/spyder/issues/4831 ?

If yes, its should be solved with spyder-3.2.1, coming soon.

**Comment by goanpeca** [comment]
Yes, I think it is the same issue, closing as a duplicate
